amqp_influxdb/__main__.py

```diff
@@ -27,6 +27,14 @@ def parse_args():
     parser = argparse.ArgumentParser()
     parser.add_argument('--amqp-hostname', required=False,
                         default='localhost')
+    parser.add_argument('--amqp-port', required=False,
```

Comment:
please remove the argument and deduct the port from whether SSL is enabled in the manager blueprint or not.